projects/core/src/cart/event/cart-event.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { StateEventModule } from '../../state/event/state-event.module';
+import { CartActions } from '../store/actions';
+import { CartEvents } from './cart-event.model';
+@NgModule({
+  imports: [
+    StateEventModule.forChild([
+      {
```

Comment:
We'd need to extend this demo with the ability to merge multiple actions to the same event
- add
- update
- remove

Should be aggregated in a single event. The event payload would have more information on the actual change, so that the consumer understands what was going one, i.e.:
- quantity was increased/decreased with x
- previous quantity was y
- item (product/service) code is ...

Perhaps we keep the separate events as well, but they seem to fine-grained for most cases.